### PR TITLE
feat(footer): add footer

### PR DESCRIPTION
**Pull request by VMBindraban**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,6 +1,231 @@
 {
   "extends": "./node_modules/aurelia-tools/.eslintrc.json",
   "rules": {
-    "no-new-func": 0
+    "accessor-pairs": "error",
+    "array-bracket-spacing": [
```
Comment (doktordirk) [inline .eslintrc.json]: are those all needed here? as you see above, we do already extend ./node_modules/aurelia-tools/.eslintrc.json,

**Review by doktordirk — APPROVED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (RWOverdijk) [reply .eslintrc.json]: Perhaps also use tools here? There are some nice strict rules here we could apply across the board. :)

Comment (VMBindraban) [reply .eslintrc.json]: Move the rules to Tools and use tools here?

Comment (RWOverdijk) [reply .eslintrc.json]: @VMBindraban Remove these rules, create a ticket with missing rules here: https://github.com/SpoonX/spoonx-tools

**Review by RWOverdijk — APPROVED** [review]

**Pull request merged**